scripts/ci-entrypoint.sh

```diff
@@ -36,144 +36,4 @@ source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 # shellcheck source=../hack/parse-prow-creds.sh
 source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 
-# build Kubernetes E2E binaries
-build_k8s() {
-    # possibly enable bazel build caching before building kubernetes
-    if [[ "${BAZEL_REMOTE_CACHE_ENABLED:-false}" == "true" ]]; then
-        create_bazel_cache_rcs.sh || true
-    fi
-
-    pushd "$(go env GOPATH)/src/k8s.io/kubernetes"
-
-    # make sure we have e2e requirements
-    bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
-
-    # ensure the e2e script will find our binaries ...
-    mkdir -p "${PWD}/_output/bin/"
-    cp -f "${PWD}/bazel-bin/test/e2e/e2e.test" "${PWD}/_output/bin/e2e.test"
-
-    PATH="$(dirname "$(find "${PWD}/bazel-bin/" -name kubectl -type f)"):${PATH}"
-    export PATH
-
-    # attempt to release some memory after building
-    sync || true
-    (echo 1 > /proc/sys/vm/drop_caches) 2>/dev/null || true
-
-    popd
-}
-
-create_cluster() {
```

Comment:
keep in mind this script is also being used by provider azure to test kubernetes on Azure and things like AzureDisk and AzureFile so we can't delete it until we have a full replacement (ie. a way to create a cluster with a CI version of k8s + skip conformance + keep the cluster around to run other tests): https://testgrid.k8s.io/provider-azure-periodic

Reply:
yes, I will not remove, just removed for now to test during the WIP, when works I will open a PR to update the periodic job.


BTW, @CecileRobertMichon I don't understand why it is trying to push the image, is something we need to do?